pandas/_libs/lib.pyx

```diff
@@ -2274,7 +2274,7 @@ def to_object_array_tuples(rows: list):
 
     k = 0
     for i in range(n):
-        tmp = len(rows[i])
+        tmp = 1 if util.is_nan(rows[i]) else len(rows[i])
```

Comment:
actually, use checknull here, ``is_nan`` is not strict enough as it doesn't check ``None`` (same below as well)

Reply:
you are right